pkg/mocks_test.go

```diff
@@ -0,0 +1,56 @@
+package pkg
```

Comment:
True, but it seems `go test` does not complain about it so :-) But yeah I agree with you on this - I just don't know where does it belong :-/